renderdoc/driver/vulkan/wrappers/vk_misc_funcs.cpp

```diff
@@ -2321,6 +2321,22 @@ VkResult WrappedVulkan::vkDebugMarkerSetObjectNameEXT(VkDevice device,
 
       Chunk *chunk = scope.Get();
 
+      data.record->LockChunks();
+      for(; data.record->HasChunks();)
```

Comment:
This loop with only a condition would be more natural as `while(data.record->HasChunks())`